test/db/cmd/cmd_pd

```diff
@@ -1915,7 +1915,7 @@ EOF
 EXPECT=<<EOF
 |           0x0001045c      STR R0, [var_8h]
 |           0x00010460      STR R1, [var_ch]
-|           0x00010464      LDR R0, [envp]
```

Comment:
Why `envp` disappeared in the test? It is a valid argument for `main`: `int main(int argc, char *argv[], char *envp[])`:
- https://www.gnu.org/software/libc/manual/html_node/Program-Arguments.html
- https://stackoverflow.com/questions/10321435/is-char-envp-as-a-third-argument-to-main-portable

Reply:
@XVilka If you look at the assembly `[fp, 8]` actually points to the 4th argument. `[fp, 4]` is what should point to `envp`